File: group-wechat/wechat-backend-001/src/main/scala/com/example/demo/controller/EchoController.scala
package com.example.demo.controller

import java.util.Collections

import org.springframework.web.bind.annotation.{GetMapping, RestController}

import scala.collection.{JavaConversions, JavaConverters}

/**
  * <p>
  * <strong>
  * 用一句话描述功能
  * </strong><br /><br />
  * 如题。
  * </p>
  *
  * @author chengchaos[as]Administrator - 2019/7/8 10:52 <br />
  * @see 【相关类方法】
  * @since 1.1.0
  */
@RestController
class EchoController {


  @GetMapping(value=Array("/_echo/hai"))
  def sayHai() : java.util.Map[String, Object] = {

    Collections.singletonMap("message", "Hai...")

  }
}

File: group-grpc/202000211/demo-client-01/src/main/java/luxe/chaos/demo/controller/EchoController.java
package luxe.chaos.demo.controller;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.web.bind.annotation.GetMapping;

import java.util.Collections;
import java.util.Map;

public class EchoController {


    private static final Logger LOGGER = LoggerFactory.getLogger(EchoController.class);


    @GetMapping(value = "/v1/say-hai")
    public Map<String, Object> sayHello() {

        return Collections.singletonMap("message", "嗨..");
    }
}
